res/database/Default/user/jorgaeff.xml

```diff
@@ -646,12 +750,14 @@
 			</effects>
 		</news>
 
-		<news id="news-jorgaeff-abstieg-02a" type="2"thread_id="news-jorgaeff-abstieg"  creator="8936" created_by="jorgaeff">
+		<news id="news-jorgaeff-abstieg-02a" type="2" thread_id="news-jorgaeff-abstieg" creator="8936" created_by="jorgaeff">
 			<title>
 				<de>SV Hamburg steigt erstmalig ab</de>
+				<en>SV Hamburg relegated for the first time</en>
 			</title>
 			<description>
 				<de>Erstmalig in seiner Vereinsgeschichte ist der SV Hamburg aus der Bundesliga abgestiegen. Er unterlag dem Glubb in einem spannenden Spiel mit 0:1 nach Verlängerung. Wackelt jetzt der Stuhl von Trainer Knut Knolle?</de>
+				<en>For the first time in its club history, SV Hamburg has been relegated from the Bundesliga. They lost to the Glubb in an exciting game with 0-1 after extra time. Is the seat of coach Knut Knolle in question now?</en>
```

Comment:
over-time?

Reply:
after extra time, AET, is the standard term.